src/com/amazon/ion/benchmark/DataConstructor.java

```diff
@@ -512,8 +521,8 @@ public static Timestamp constructTimestamp(Map<String, ReparsedConstraint> const
         Range range = DEFAULT_TIMESTAMP_IN_MILLIS_DECIMAL_RANGE;
         // Preset the local offset.
         Integer localOffset = localOffset(random);
-        // Preset the default precision.
-        Timestamp.Precision precision = PRECISIONS[random.nextInt(PRECISIONS.length)];
+        // Preset the default precision as 'Day'.
+        Timestamp.Precision precision = PRECISIONS[2];
```

Comment:
`= Timestamp.Precision.DAY;` would be simpler.

Reply:
> `= Timestamp.Precision.DAY;` would be simpler.

This will be updated in the next PR.